taraxa/state/dpos/precompiled/dpos_contract.go

```diff
@@ -876,6 +911,24 @@ func (self *Contract) state_put(key *common.Hash, state *State) {
 	}
 }
 
+func (self *Contract) check_validator_owner(owner, validator *common.Address) bool {
```

Comment:
also this function should be part of validators struct. We use different styles, shich shuld be united but Validators class use camelcase so lets keep at least classes consisstent